py/tracks/latest.toml

```diff
@@ -13,6 +13,7 @@ fixtures = [
 queries = [
     "../specs/group_by.toml",
     "../specs/like.toml",
+    "../specs/regex.toml",
```

Comment:
@mfussenegger Could you please explain? 
For regex.toml I've created a new SQL: sql/uservisits_analyzer.sql because I used an analyzer: `INDEX uagent_idx USING FULLTEXT("UserAgent") with (analyzer='english')`

Do you think I could use this index in uservisits.sql and get rid of this new file?